CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # New Relic Ruby Agent Release Notes
 
+## dev
+
+Version <dev> fixes a bug that would cause an error and a corresponding expected HTTP status code to negatively impact a transaction's Apdex.
```

Comment:
the way this sentence is worded to me sounds like its only about errors related to status codes, rather than all expected errors. My understanding is this is more about expected errors in general than the http status codes config (correct me if i'm wrong of course!). maybe this could be reworded for clarity?
something like this maybe?
```suggestion
Version <dev> fixes a bug that would cause an expected error to negatively impact a transaction's Apdex.
```

Reply:
Great call, @tannalynn. The fix includes routing the "should this error impact apdex?" check through a complete "is the error expected?" check, which will indeed check status, class, and message. But I got tunnel vision when it came to writing the text description.